docs/dev/development/index.rst

```diff
@@ -38,6 +38,7 @@ Get involved or find help using:
     cloud
     email
     token-scanning
+    attestation-internals
```

Comment:
This will need to move as well.

Reply:
Hmm, do you want it moved to the user docs? I think I put it there originally, but moved it to dev-docs after an earlier round of review.

xref: https://github.com/pypi/warehouse/pull/16063#discussion_r1674449662

Reply:
Whoops, just saw the other comment that got folded (https://github.com/pypi/warehouse/pull/16398/files#r1831458028). I'll move to security!